ui/src/dashboards/components/DashboardPage.tsx

```diff
@@ -46,6 +49,18 @@ const dashRoute = `/${ORGS}/${ORG_ID}/${DASHBOARDS}/${DASHBOARD_ID}`
 
 @ErrorHandling
 class DashboardPage extends Component<Props> {
+  public componentDidMount() {
+    const {dashboard, setMountID} = this.props
+    const mountID = uuid.v4()
+    setMountID('dashboard', mountID)
+
+    event('Dashboard Mounted', {
+      mountID,
```

Comment:
I think mountID should be in the third argument to event, since it is unbounded in cardinality
orgID is already in the reporting context so you don't need to add it here!  (`SetOrg.tsx has       updateReportingContext({orgID: orgID})`)